os/drivers/seclink/seclink_drv_utils.c

```diff
@@ -0,0 +1,46 @@
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <errno.h>
+#include "seclink_drv_utils.h"
+
+#define NSPERSEC 1000000000
+#include <fcntl.h>
+#include <tinyara/timer.h>
+// microsecnd
+
+void get_frt_time(struct timer_status_s *ts)
+{
+	struct timer_status_s before;
+    struct timer_status_s after;
+    int frt_fd;
+
+    frt_fd = open("/dev/timer0", O_RDONLY);
+    if (frt_fd < 0) {
+        lldbg("ERROR: Failed to open, errno %d\n", errno);
+        return;
+    }
+
+    ioctl(frt_fd, TCIOC_SETMODE, MODE_FREERUN);
+    ioctl(frt_fd, TCIOC_START, 0);
+    ioctl(frt_fd, TCIOC_GETSTATUS, (unsigned long)(uintptr_t)ts);
+    close(frt_fd);
+}
+
+uint32_t sdrv_calc_elasped(struct timespec *start, struct timespec *end)
+{
+	uint32_t elapsed;
+	printf("[pkbuild] start %u %u end %u %u\n", start->tv_sec, start->tv_nsec, end->tv_sec, end->tv_nsec);
+	if (start->tv_nsec > end->tv_nsec) {
+		elapsed = NSPERSEC - start->tv_nsec + end->tv_nsec;
+		elapsed += ((end->tv_sec - 1 - start->tv_sec) * NSPERSEC);
+	} else {
+		elapsed = end->tv_nsec - start->tv_nsec;
+		elapsed += (end->tv_sec - start->tv_sec) * NSPERSEC;
+	}
+	return elapsed;
+}
+
+uint32_t sdrv_calc_elapsed2(struct timer_status_s *start, struct timer_status_s *end)
```

Comment:
I think naming with number is function2, function2 is not good.
Could you rename them to names that matches the actual behavior?
For example, `sdrv_cal_elasped_timespec` for `sdrv_calc_elasped_timer_status` and `` for `sdrv_calc_elapsed2`?